### PR TITLE
Issue/85

### PR DESCRIPTION
**Pull request by minborg**
This PR fixes a nasty concurrency bug

### CODE REVIEW

File: src/test/java/mytest/RuntimeCompileTest.java [hunk 1]
```diff
@@ -64,6 +64,7 @@ public void testMultiThread() throws Exception {
         final ClassLoader cl = new URLClassLoader(new URL[0]);
         final CachedCompiler cc = new CachedCompiler(null, null);
         final int nThreads = Runtime.getRuntime().availableProcessors();
+        System.out.println("nThreads = " + nThreads);
```
Comment (minborg) [inline src/test/java/mytest/RuntimeCompileTest.java:67]: Interesting to know, if run under a Docker agent

**Review by RobAustin — APPROVED** [review]

**Review by alamar — APPROVED** [review]

File: src/test/java/mytest/RuntimeCompileTest.java [hunk 2]
```diff
@@ -42,7 +42,7 @@ public void outOfBounds() throws Exception {
         }
     }
 
-    @Ignore("see https://teamcity.chronicle.software/viewLog.html?buildId=639347&tab=buildResultsDiv&buildTypeId=OpenHFT_BuildAll_BuildJava11compileJava11")
+    //@Ignore("see https://teamcity.chronicle.software/viewLog.html?buildId=639347&tab=buildResultsDiv&buildTypeId=OpenHFT_BuildAll_BuildJava11compileJava11")
```
Comment (alamar) [inline src/test/java/mytest/RuntimeCompileTest.java:45]: Maybe we can delete this line, it will become obsolete eventually

Comment (minborg) [reply src/test/java/mytest/RuntimeCompileTest.java:45]: Yepp. I merged before I read this... Sorry.

**Review by JerryShea — COMMENTED** [review]
great catch

**Pull request merged**